### ISSUE TITLE
Error Code 2(Assertion vol ==1 || vol == params.nbOutputmaps.failed) failure of TensorRT 8.6.1.6 when running trtexec on GPU RTX3080

### ISSUE THREAD

**Issue by qinyage** [NVIDIA/TensorRT#3458]
## Description

I want to use tensorRT to accelerate and quantize my model on RTX3080, but when quantizing in int8 precision, there is a problem that a lot of network layers are still in fp32 precision, so I try to process my model by pytorch-quantization and get a new onnx model，but I get the following error when I accelerate this new onnx model with trtexec。
![a0ccb167c074abdaaf0d921427ce150](https://github.com/NVIDIA/TensorRT/assets/48047735/6517cb2a-78f6-4665-859f-9c43ff5c54ea)

## Environment

**TensorRT Version**: 8.6.1.6

**NVIDIA GPU**: RTX3080

**NVIDIA Driver Version**: 

**CUDA Version**: 11.6

**CUDNN Version**:

Operating System:window11

Python Version (if applicable):3.8.15

Tensorflow Version (if applicable):

PyTorch Version (if applicable):1.13.0

Baremetal or Container (if so, version):

## Relevant Files

**Model link**:
[](url)

## Steps To Reproduce

[log.txt](https://github.com/NVIDIA/TensorRT/files/13364954/log.txt)

**Comment by zerollzeng** [comment]
1. Does this onnx model works for onnxruntime? You can check this by `polygraphy run model.onnx --onnxrt`
2. If yes for 1), could you please try latest 9.1 release?

Thanks!

**Comment by qinyage** [comment]
> 1. Does this onnx model works for onnxruntime? You can check this by `polygraphy run model.onnx --onnxrt`
> 2. If yes for 1), could you please try latest 9.1 release?
> 
> Thank you very much for your reply and suggestion, my model is a lightweight model, the problem appeared when I inserted fake quantization nodes into each layer and did not go through finetune, then the problem disappeared when I inserted fake quantization nodes into only one of the layers, so I guess the previous fake quantization nodes were too destructive to the accuracy of the model, which led to the failure of the tensorRT.

**Comment by zerollzeng** [comment]
Okay, can we close this? Thanks!
